ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml

```diff
@@ -0,0 +1,61 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |-
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.14 AS builder
+    WORKDIR /build
+    COPY . .
+    RUN export CGO_ENABLED=0 make build
+
+    FROM registry.ci.openshift.org/ocp/4.14:base
+    COPY --from=builder build/bin/manager /usr/bin/operator-controller-manager
+    LABEL io.k8s.display-name="OpenShift Operator Lifecycle Manager Operator Controller" \
+          io.k8s.description="This is a component of OpenShift Container Platform that allows operator installation."
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.19-openshift-4.14:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.14
+  to: olm-operator-controller
+promotion:
+  name: "4.14"
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.14"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.14"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: make test-unit
+  container:
+    from: src
+- as: vet
+  commands: make vet
+  container:
+    from: src
+- as: fmt
+  commands: make fmt
+  container:
+    from: src
+zz_generated_metadata:
```

Comment:
Please add the verify-deps target à la https://github.com/openshift/release/blob/a029cc215f57d17a9e38bb30ec427127cc042067/ci-operator/config/openshift/operator-framework-catalogd/openshift-operator-framework-catalogd-main.yaml#L64

Reply:
Roger that.  I was going to wait until I had the existing tests running, to iterate through the cross-fittings, but I've added it now.